### ISSUE TITLE
Using cx_Freeze package the program for dragging and uploading files written in pyside6, but cannot drag and drop files

### ISSUE THREAD

**Issue by zhenzi0322** [marcelotduarte/cx_Freeze#1910]
I have a simple drag and drop file processing program, but using cx_ Why does Freeze no longer support drag and drop after packaging

`app.py`
```python
import sys
from PySide6.QtWidgets import QApplication, QMainWindow, QTextBrowser


class Window(QMainWindow):
    def __init__(self):
        super(Window, self).__init__()
        self.paths = ""
        self.setWindowTitle('文件拖入')
        self.resize(600, 400)
        self.textBrowser = QTextBrowser()
        self.setCentralWidget(self.textBrowser)
        self.setAcceptDrops(True)

    def dragEnterEvent(self, event):
        self.setWindowTitle('dragEnterEvent')
        file = event.mimeData().urls()[0].toLocalFile()
        if file not in self.paths:
            self.paths += file + "\n"
            self.textBrowser.setText(self.paths)
            event.accept()

if __name__ == "__main__":
    app = QApplication(sys.argv)
    window = Window()
    window.show()
    sys.exit(app.exec())
```

`setup.py`
```python
from cx_Freeze import setup, Executable

setup(
    name='test',
    author='xxxx',
    author_email='xxx@xxx.com',
    version='1.0',
    description="xxxx",
    license='MIT',
    executables=[
        Executable(
            "app.py",
            base="win32gui",
            target_name="app",
            uac_admin=True
        )
    ],
)
```

```
python setup.py build
```

**Comment by marcelotduarte** [comment]
Works without uac_admin. After the freeze, the behavior is the same as in Python. Adding uac_admin changes the behavior.

**Comment by marcelotduarte** [comment]
Can I close this issue?

**Comment by marcelotduarte** [comment]
Can you test the new uac_uiaccess (implemented in PR #2135)?
add uac_uiaccess=True 

You can test the patch in the latest development build:
`pip install --force --no-cache --pre --extra-index-url https://marcelotduarte.github.io/packages/ cx_Freeze`
For conda-forge the command is:
`conda install -y --no-channel-priority -S -c https://marcelotduarte.github.io/packages/conda cx_Freeze`

**Comment by marcelotduarte** [comment]
Closing due to lack of response.
